sources/filesystem/__init__.py

```diff
@@ -0,0 +1,138 @@
+"""Those resources collect filepaths from local folder or Google Drive folder to destinations"""
+import logging
+from pathlib import Path
+from typing import Optional, Sequence, Dict, Any
+
+import dlt
+from dlt.extract.source import TDataItem
+
+from .helpers import build_service, download_file_from_google_drive, get_files_uris
+from .settings import (
+    AUTHORIZED_USER_PATH,
+    DATA_DIR,
+    FOLDER_IDS,
+    STORAGE_FOLDER_PATH,
+    ClIENT_SECRET_PATH,
+)
+
+
+@dlt.resource(write_disposition="replace")
+def local_folder(
+    extensions: Optional[Sequence[str]] = None,
```

Comment:
we'll convert it into fsspec destination so we can also read from buckets. to make it production grade:
- should descend into folders
- should do incremental loading using "modified" timestamp on the file
- should delete processed files as an option (this one is tricky but doable transactionally)